proto/resource_manager.proto

```diff
@@ -188,13 +188,14 @@ message Error {
 }
 
 enum RunawayAction {
-  DryRun = 0;    // do nothing
-  CoolDown = 1;  // deprioritize the task 
-  Kill = 2;      // kill the task
+  NoneAction = 0;
+  DryRun = 1;    // do nothing
+  CoolDown = 2;  // deprioritize the task 
+  Kill = 3;      // kill the task
 }
 
 enum RunawayWatchType {
-  None = 0;
+  NoneWatch = 0;
```

Comment:
NIT: I feed None is better, no need to repeat the watch since it's the type of watch.